tensorflow_datasets/datasets/amazon_us_reviews/amazon_us_reviews_dataset_builder.py

```diff
@@ -170,18 +132,25 @@ def _generate_examples(self, file_path):
     """Generate features given the directory path.
 
     Args:
-      file_path: path where the tsv file is stored
+      file_path: path where the json file is stored
 
     Yields:
       The features.
     """
-
-    with epath.Path(file_path).open() as tsvfile:
-      # Need to disable quoting - as dataset contains invalid double quotes.
-      reader = csv.DictReader(
-          tsvfile, dialect="excel-tab", quoting=csv.QUOTE_NONE
-      )
-      for i, row in enumerate(reader):
-        yield i, {
-            "data": row,
-        }
+    with open(file_path, "r", encoding="utf-8") as json_file:
+        for i, row in enumerate(json_file):
+            example = json.loads(row)
+            yield i, {
+                "unixReviewTime": int(example["unixReviewTime"]),
+                "reviewTime": example["reviewTime"],
+                "reviewerID": example["reviewerID"],
+                "reviewerName": example["reviewerName"],
+                "asin": example["asin"],
+                "overall": int(example["overall"]),
+                "summary": example["summary"],
+                "reviewText": example["reviewText"],
+                "verified": example.get("verified", False),
+                "style": str(example.get("style", {})),
+                "vote": int(example.get("vote", 0)),
+                "image": str(example.get("image", "")),
```

Comment:
@marcenacp 
the `image` column (nullable) contains a comma-separated list of image URLs. Is it a good idea to convert the whole to a string? is there any better approach?

Reply:
What about using a `Sequence` of str: https://www.tensorflow.org/datasets/api_docs/python/tfds/features/Sequence?

(See for instance usage in https://github.com/tensorflow/datasets/blob/master/tensorflow_datasets/text/gem/gem.py)